lua/entities/gmod_wire_expression2/core/string.lua

```diff
@@ -36,6 +36,70 @@ end)
 
 /******************************************************************************/
 
+registerOperator("fea", "nss", "", function(self, args)
+	local keyname, valname = args[2], args[3]
+	local str = args[4]
+	str = str[1](self, str)
+
+	local statement = args[5]
+
+	for key=1, #str do
+		local value = str[key]
```

Comment:
Use string.sub here instead of string[k] since that's garry's slow implementation https://github.com/Facepunch/garrysmod/blob/4aefc7d7f468998cd3239c0baa560a389f27d884/garrysmod/lua/includes/extensions/string.lua#L291

Reply:
Fixed, I have once again assumed Garry is a good programmer